internal/openchoreo-api/mcphandlers/organizations.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2025 The OpenChoreo Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package mcphandlers
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/openchoreo/openchoreo/internal/openchoreo-api/services"
+)
+
+type MCPHandler struct {
+	Services *services.Services
+}
+
+func (h *MCPHandler) GetOrganization(name string) (string, error) {
+	ctx := context.Background()
```

Comment:
Shouldn't this be the context coming from the HTTP request?

Reply:
Updated to use the context coming from the MCP session (initiated by the SDK)